src/components/App/index.tsx

```diff
@@ -38,6 +39,7 @@ export type Profile = {
 };
 
 type State = {
+  loading: boolean;
```

Comment:
PR #136 removed the state entirely from this component. I think maybe we should put this `loading` flag in the redux store, rather than adding back in a component state. What do you think?

Reply:
We could probably now that #136 has landed.